Lib/test/test_enum.py

```diff
@@ -1569,6 +1569,17 @@ class IntFlag1(IntFlag):
         self.assertIn(IntEnum1.X, IntFlag1)
         self.assertIn(IntFlag1.X, IntEnum1)
 
+    def test_contains_does_not_call_missing(self):
+        class AnEnum(Enum):
+            UNKNOWN = None
+            LUCKY = 3
+            @classmethod
+            def _missing_(cls, *values):
+                return cls.UNKNOWN
+        self.assertTrue(None in AnEnum)
+        self.assertTrue(3 in AnEnum)
+        self.assertFalse(7 in AnEnum)
```

Comment:
Maybe we should test that we don't create members as well?

Reply:
Only `Flag`s might create new members, and that's intentional (no point in doing the work twice).

Reply:
Oh yes, boundaries are only relevant to flags.